package/yast2-iscsi-client.changes

```diff
@@ -1,3 +1,10 @@
+-------------------------------------------------------------------
+Wed Jan 29 14:26:18 UTC 2020 - Ancor Gonzalez Sosa <ancor@suse.com>
+
+- Backport of Knut changes in 4.2.3 to fix bsc#1160374: keep the
+  current state after writing the configuration by default.
+- 4.1.9
```

Comment:
MMm I miss a mention to the bug it is fixing: bsc#1162078

https://bugzilla.suse.com/show_bug.cgi?id=1162078

Reply:
True

Reply:
Fixed via `git commit --amend`